specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -232,17 +232,18 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
           isSaveDisabled &&
           showCarry &&
           showBulkCarry ? (
-            <Label.Inline>
-              <Input.Generic
-                type="number"
-                value={carryForwardAmount}
-                onValueChange={(value): void =>
-                  carryForwardAmount === undefined
-                    ? setCarryForwardAmount(1)
-                    : setCarryForwardAmount(Number.parseInt(value))
-                }
-              />
-            </Label.Inline>
+            <Input.Integer
+              aria-label={formsText.bulkCarryForwardCount()}
+              className="!w-fit"
+              min={1}
+              placeholder="1"
```

Comment:
hmm, placeholder of 1 is still not clear what this field is about
could you either set placeholder to `formsText.bulkCarryForwardCount()`, or add a `title={formsText.bulkCarryForwardCount()}`
edit: I have just noticed that there is a button visible right after the field - that makes it more clear what the field is about so the way it is now is ok!